<launch>
  <!-- Namespace for the node and services -->
  <arg name="ns"  default="gqcnn" />
  
  <!-- Configuration file for Grasp Planner Node -->
  <arg name="config"           default="$(find gqcnn)/cfg/ros_nodes/grasp_planner_node.yaml" />
  <arg name="model_dir"        default="$(find gqcnn)/models/GQCNN-2.0" />
  
  <node name="GQCNN_grasp_planner"  pkg="gqcnn" type="grasp_planner_node.py" output="screen" >
    	<param name="config"           value="$(arg config)" />
    	<param name="model_dir"           value="$(arg model_dir)" />
  </node>
</launch>